tests/behat/features/revoke.feature

```diff
@@ -0,0 +1,31 @@
+@retry
+
+Feature: See other devices and revoke their access
+  As a CMS user
+  I want to see other devices that are currently logged in
+  So that I can revoke their access
+
+  Background:
+    Given I am logged in with "ADMIN" permissions
+    # Create a mock login session
+    And There is a login session for a second device
+
+  Scenario: I can see other devices and revoke their access
+    When I go to "/admin/security"
+    # Click the ADMIN user
+    And I click the ".col-FirstName" element
+    # Ensure XHR loaded from endpoint
+    And I wait for 2 seconds
```

Comment:
There's a few [I wait until I see](https://github.com/silverstripe/silverstripe-behat-extension/blob/fe0818afc293d4c708d9cb075871fe9267d6326f/src/Context/BasicContext.php#L1111-L1158) operations you could use here.

Reply:
Updated